docs/changes.rst

```diff
@@ -85,7 +85,14 @@ Change log
   :attr:`Panel.scripts <debug_toolbar.panels.Panel.scripts>` property.
 * Removed support for end of life Django 1.11. The minimum supported Django is
   now 2.2.
-
+* JavaScript is now loaded as a `JavaScript module`_. When developing using the
+  Django development server, this will normally load fine and not present an
```

Comment:
This is not correct, CORS is required in any scenario where static files are served under a different domain, development or not.

Reply:
The Django development server is under one origin: `localhost:8000`. What you're describing is the 2nd part of the sentence "if your application server and static files server are at different locations".

Suggestions on wording?

Reply:
How about:

```rst
JavaScript is now loaded as a `JavaScript module`_. If your application server
and static files server are at different origins (e.g. different host/port),
you may see CORS errors in your browser's development console. See the
"Cross-Origin Request Blocked" section of the :doc:`installation docs
<installation>` for details on how to resolve.
```

There's no need to mention development mode since it's not really relevant; only the origin is relevant.

Reply:
There is no mention of "development mode", perhaps this could be clearer. There is only a mention of the development server (aka `runserver`). I want to make clear to users that have a very standard environment -- e.g. the one outline in  the Django tutorial and docs -- that no action is necessary.

Reply:
> There is no mention of "development mode", perhaps this could be clearer.

It's not relevant at all. Why mention something that's not relevant? It doesn't matter if someone is using `runserver`, `daphne`, or anything else. It doesn't matter if they've followed the django tutorial, or have a production-ready setup.

The only factor that plays any part in this new requirements is if the origins are different.

Reply:
I think it is worthwhile to reassure users (especially new users) that are using a very typical setup that no additional action is required. Advanced users using an advanced setup are more likely to understand the nuances of different origins and CORS.

Reply:
> I think it is worthwhile to reassure users (especially new users) that are using a very typical setup that no additional action is required. Advanced users using an advanced setup are more likely to understand the nuances of different origins and CORS.

Agreed, but I'm not sure the change log is read by new users. I think something closer to what @WhyNotHugo suggested would be better. Maybe we can highlight the happy flow case, then follow-up with the exception?

    The Debug Toolbar loads a `JavaScript module`_.  Typical local 
    machine development is not be impacted. However, if the
    application and static files are served from different hosts...

Reply:
Thanks for the feedback, all. I've tried to clean this up a little from the provided feedback.